### ISSUE TITLE
Circles are created with diameter instead of radius

### ISSUE THREAD

**Issue by wasnotrice** [shoes/shoes-deprecated#199]
The manual says that when you create a circle like this:

```
oval(100, 100, 200)
```

you should get a circle with top left (100, 100), and a radius of 200. You actually get a circle with _diameter_ 200. Here's a demo app:

https://gist.github.com/2341166

Red Shoes:

![Red Shoes](https://img.skitch.com/20120409-8jad2m92cpf5as51928neyg793.medium.jpg)

Purple Shoes:

![Purple Shoes](https://img.skitch.com/20120409-g9dtr7ka8mka655a5ujbk9r55q.medium.jpg)

**Comment by nashby** [comment]
I'm not sure but it can be because of this line https://github.com/shoes/shoes/blob/develop/lib/shoes/chipmunk.rb#L18
I'll try to fix it tonight.

**Comment by ashbb** [comment]
Yeah, this is a bad spec, may be a bug of Red Shoes, IMHO. So, I decided to change the behavior in Green and Purple Shoes.

@nashby 
It's not  because of chipmunk.rb. Look at this: https://github.com/shoes/shoes/blob/develop/shoes/canvas.c#L508
In Red Shoes, radius seems to be set to width.

**Comment by wasnotrice** [comment]
@ashbb I agree, this is a bug in red shoes. Green and Purple are correct. To not break current apps, it might be better to leave this behavior as is in Red for now, and fix it in a future release. But the manual is confusing.
